addon/services/tour.js

```diff
@@ -449,16 +449,19 @@ export default Service.extend(Evented, {
       const currentStep = tour.steps[index];
 
       currentStep.on('before-show', () => {
-        if (this.get('modal')) {
-          const currentElement = this.getElementForStep(currentStep);
-
-          if (currentElement) {
+        const currentElement = this.getElementForStep(currentStep);
+        if (currentElement) {
+          if (this.get('modal')) {
```

Comment:
It would be nice to deduplicate the `addClass` calls. I'll take a stab at that reorganization :+1: